blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/formats/JSONMetricsContainer.java

```diff
@@ -87,7 +92,7 @@ public boolean areDelayedMetricsPresent() {
     public static class JSONMetric {
         private String metricName;
         private Object metricValue;
-        private long collectionTime;
+        private Long collectionTime = null;
```

Comment:
If you change that to "Long", the following existing code might break. If collectionTime is null, I am guessing it will cause NullpointerException.

``
public long getCollectionTime() {
            return collectionTime;
        }
``